modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DELinkToPageField.testcase

```diff
@@ -81,12 +81,45 @@ definition {
 	}
 
 	@description = "This is a stubbed description"
-	@ignore = "true"
 	@priority = "5"
 	test CanClearInput {
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Test Page Name");
 
-		// TODO LRQA-70110	Refactor of Renderer Link to Page Field
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(structureName = "WC Structure Title");
+
+		DEBuilder.addField(
+			fieldLabel = "Link to Page",
+			fieldName = "Link to Page");
+
+		WebContentStructures.saveCP();
+
+		NavItem.gotoWebContent();
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Title");
 
+		PortletEntry.inputTitle(title = "Web Content Title");
+
+		DERenderer.inputDataInLinkToPageField(
+			webContentLinkToPage = "Test Page Name",
+			webContentLinkToPageFieldLabel = "Link to Page");
+
+		Button.clickPublish();
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
+
+		DERenderer.clearImageField(fieldLabel = "Link to Page");
```

Comment:
This line will be affected by this [pull](https://github.com/liferay-forms/liferay-portal/pull/956), specifically by this [commit](https://github.com/liferay-forms/liferay-portal/pull/956/commits/ff7e18cb849b3a6804f85974d751914fbc4eb5ef). Please make sure to rebase against master to fix this problem.